pkg/client/cache/listwatch_test.go

```diff
@@ -117,7 +117,7 @@ func TestListWatchesCanWatch(t *testing.T) {
 		{
 			location: buildLocation(
 				testapi.Default.ResourcePathWithPrefix("watch", "nodes", api.NamespaceAll, ""),
-				buildQueryValues(url.Values{"resourceVersion": []string{""}})),
+				buildQueryValues(url.Values{})),
```

Comment:
Yeah - what I wanted to say is that we want a consistent behavior if you call:
c.Foo(ns).Watch(options)
and
listwatch.Watch(options)

Regarding getting rid of buildQueryValues - I'm happy to do it, but I will do it in a separate PR.